contrib/babelfishpg_tsql/src/pltsql_coerce.c

```diff
@@ -301,6 +301,8 @@ tsql_cast_raw_info_t tsql_cast_raw_infos[] =
 	{TSQL_CAST_WITHOUT_FUNC_ENTRY, "sys", "uniqueidentifier", "sys", "bpchar", NULL, 'i', 'i'},
 	{TSQL_CAST_WITHOUT_FUNC_ENTRY, "sys", "uniqueidentifier", "pg_catalog", "varchar", NULL, 'i', 'i'},
 	{TSQL_CAST_WITHOUT_FUNC_ENTRY, "sys", "uniqueidentifier", "sys", "varchar", NULL, 'i', 'i'},
+/*  bit -> string via I/O */
```

Comment:
Can we add few more tests for this cast exclusively , cause I dont think we have enough coverage here.

Reply:
Added